src/test/java/utilities/Util.java

```diff
@@ -1,6 +1,25 @@
 package utilities;
 
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
 public class Util {
+	/*
+	 * Checks if an element is displayed
+	 * 
+	 * @Author: Nilupar Erkin
+	 */
+	
+	public static boolean isElementDisplayed(WebDriver driver, String xPathOfElement) {
```

Comment:
It would be a better idea to use By object instead of xpath, would be more reusable